CROSSSECTOR/DIH/DigitalInnovationHubService/examples/example.json

```diff
@@ -0,0 +1,35 @@
+{
+	"id": "b6IZuH0B_X_d5NJkB0eY",
+	"type": "DigitalInnovationHubService",
+	"title": "Trend watching",
+	"description": "Provision of insights and trend on markets to companies to stimulate their internal innovation",
+	"serviceImage": {
+		"base64Data": "iVBORa1ytHEAAAAAElFTkSuQmCC",
+		"filename": "it.png"
+	},
+	"category": "Ecosystem",
+	"serviceType": "DIH Innovation Development",
+	"service": "Trend watching",
+	"target": [
+		"Engineers",
+		"Directors"
+	],
+	"url": "https://www.sample-dih.com/trend-watching",
+	"additionalmaterial": [{
+		"label": "Brochure",
+		"url": "https://www.sample-dih.com/trend-watching/brochure.pdf"
+	}],
+	"contacts": [{
+		"name": "Mark Johnson",
+		"email": "mark.johnson@sample-dih.it"
+	}],
+	"author": "John Doe",
+	"relation": {
+		"dih": [{
+			"to": "bqIXuH0B_X_d5NJkBkeO"
+		}]
+	},
+	"extra": {

```

Comment:
Bad idea subproperty named 'properties' which is a reserved word for json schema

Reply:
you need a name for subproperty